pulumi/folio/kafka.ts

```diff
@@ -1,14 +1,15 @@
 import * as eks from "@pulumi/eks";
 import * as k8s from "@pulumi/kubernetes";
+import * as pulumi from "@pulumi/pulumi";
 
 // TODO Use a chart from a URL rather than a repo so that the user doesn't need the chart on their local machine.
 
 // Deploy Kafka using the Helm chart.
-export module deployment {
-    export function helm(cluster: eks.Cluster, appNamespace: k8s.core.v1.Namespace) {
```

Comment:
Is this incorrect? I want to understand type better

Reply:
Just suggesting it from a noun to a verb which is what I had in mine to make it more consistent.

Reply:
Sorry, I meant the `k8s.core.v1.Namespace` to `pulumi.Output<string>`

Reply:
Oh, right. I don't remember changing the params. I don't think there is anything wrong with passing in the whole namespace object.